### PR TITLE
Content revamp and suggestions for Installation pages

### PR DESCRIPTION
**Pull request by divyk**
Installation page revamp: 
Intended Audience: Beginner Developers and Intermediate Developers 
Phase 1:
Added complete and verified steps for installations for: 
- Ubuntu
- Windows
- Macintosh 
- CLI 
Each operating system has a separate page for installation

Context: 
These pages are written from scratch and the installation steps have been verified to run flawlessly. 
Earlier, there was a single page for installation which was said to lack necessary steps.
 
Phase 2: Addition of gifs and other visuals.
Attaching the ToC for your reference: 

image

With Regards, 
Nawab & Divya

### CODE REVIEW

**Comment by divyk** [comment]
@journey2w @mijin85cho , please review and give your valuable suggestions.

File: docs/application/tizen-studio/setup/install-ubuntu.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+- Uninstalling -->
+
+## Pre-Installation Check 
+To successfully install Tizen Studio, ensure all the prerequisites are met.
+
+### Operating System Prerequisites
+Ensure that the target system meets the following operating system requirements:
+<table>
+  <tr>
+    <th colspan="2">Module </th>
```
Comment (mijin85cho) [inline docs/application/tizen-studio/setup/install-ubuntu.md]: The page separation was too rough. This page is for Ubuntu, but Prerequisites information for Windows and MacOs are included.

Same issue to pages for macOS and Windows, too.

File: docs/application/tizen-studio/setup/install-ubuntu.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+- `sudo apt-get install libudev-dev\`
+- `sudo apt-get install libsdl1.2debian\`
+- `sudo apt-get install bridge-utils\`
+- `sudo apt-get install openvpn\`
+
+## Launch Installer 
+Following steps help you to launch the installer:
+
+1. Open your terminal and execute the following command:
+	- `chmod +x web-ide_Tizen_Studio_3.1_ubuntu-64.bin`
```
Comment (mijin85cho) [inline docs/application/tizen-studio/setup/install-ubuntu.md]: If you want to write the version information please try to follow up the latest version always. For now, Tizen Studio 3.2 is the latest

**Comment by divyk** [comment]
Closing this PR, will reopen with more content and details. Thanks.

**Comment by nawab-samsung** [comment]
@mijin85cho, thanks for your feedback. 
As per yesterday's communication, we will be re-raising the PRs as per the discussed plan. 
I will be taking care of all the concerns mentioned herein. 
Thanks,

File: docs/application/tizen-studio/setup/tizen-cli.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+## Install Tizen Studio Using CLI
```
[the remark this answers is not in the document]
Comment (nawab-samsung) [reply docs/application/tizen-studio/setup/tizen-cli.md]: Will take care of this concern. Will intimate you with restructuring the ToC

**Comment by mijin85cho** [comment]
[Back Up] Answer from nawab by e-mail.
@nawab-samsung As I told you, please use Github to communicate instead of using several ways....
Use one channel to communicate, not to scatter the history.


--------------

Hi Catherine, 

I am quoting your concern regarding the installation pages, and providing reply inline. 

Thank you for your great suggestions. 

> Actually I'm not sure the effectivity of separating the pages in this way as I told you.


 Three installation pages are needed as:

-  the steps to install JDK (after Oracle JDK licensing issue, open JDK was introduced) so configuring open JDK differs for each operating system 

- The mode of installation for each OS is different, 

   - In Ubuntu, we need to use  terminal to install dependencies etc.,  while as in windows we dont need to similarly with Mac

> I understand you want to suggest each page for each OS, but as you can see many steps are overlapped

- There may be probable overlap in prerequisites section, because each page is a complete page and should be treated seperately


> (Even in this PR we have information for 3 OSs in each pages.)

 Shall make it specific to each OS

- Will make the pages completely OS specific.


> I want to suggest you to check the format of https://docs.tizen.org/application/dotnet/samples/account/overview or https://docs.tizen.org/application/dotnet/api/xamarin-forms-limitations.
> Same relevant pages on TD are https://developer.tizen.org/development/sample/.net-application/account and https://developer.tizen.org/development/api-reference/.net-application/current-xamarin.forms-limitations.

 There is use of CSS in the first page, if you guide us with the layout, certainly this will be a good choice 

> In these pages, you can see that we have taps for Mobile/TV/Wearable, so a user can click the desired profile to see the only the sample apps or limitation info for "that" profile.

 There is use of CSS in the first page(https://docs.tizen.org/application/dotnet/samples/account/overview) , if you guide us with the layout, certainly this will be a good choice


> How about using this from to these pages, too?
> 
> Also brief comments from the quick review..

Your valuable comments  taken and shall be applied with utmost care on quality.  

Thanks, 

With Regards, 

Nawab

**Pull request closed without merging**